plugins/techdocs/src/reader/transformers/addGitFeedbackLink.ts

```diff
@@ -34,21 +37,13 @@ export const addGitFeedbackLink = (configApi: any): Transformer => {
     let gitHost = '';
 
     const sourceURL = new URL(sourceAnchor.href);
-    const githubHosts = configApi
-      .getConfigArray('integrations.github')
-      .map((integration: any) => integration.data.host);
-    const gitlabHosts = configApi
-      .getConfigArray('integrations.gitlab')
-      .map((integration: any) => integration.data.host);
+    const integration = scmIntegrationsApi.byUrl(sourceURL);
 
     // don't show if can't identify edit link hostname as a gitlab/github hosting
-    if (
-      githubHosts.includes(sourceURL.hostname) ||
-      sourceURL.origin.includes('github')
-    ) {
+    if (integration?.type === 'github' || sourceURL.origin.includes('github')) {
```

Comment:
should be able to drop the second check now, since reading from the integrations API will always include `github.com`. Same goes for GitLab below

Reply:
According to tests, the edit link should be available even if there is no matching integration config, if the source URL match github/gitlab, as I understood them?

Reply:
Eh, yeah but that's also prolly a bit on and a weird assumption, doesn't hold true for out instance anyway :grin:

I'm thinking remove the check and the tests that expect it, agree with that @backstage/techdocs-core?

Reply:
Agree. Not sure if users will be affected by that though? I don't remember why it was added in the first place, because it sure is a bit of an odd assumption, looking back

Reply:
@erikxiv Mind removing the second check? After that this is definitely ready to merge

Reply:
Not at all. Pushed.